commit 2ddecc23862bab1a9a9e51e097aefc92ec305e28
Author: Marius Wachtler <malloc@inode.at>
Date:   Tue Oct 12 09:09:07 2010 +0200

    Fix a small bug in XPath evaluation code

diff --git a/xpath.c b/xpath.c
index 8482c40..b24ca69 100644
--- a/xpath.c
+++ b/xpath.c
@@ -12655,7 +12655,7 @@ xmlXPathCompOpEvalFirst(xmlXPathParserContextPtr ctxt,
             return (total);
 #ifdef XP_OPTIMIZED_FILTER_FIRST
 	case XPATH_OP_FILTER:
-                total =+ xmlXPathCompOpEvalFilterFirst(ctxt, op, first);
+                total += xmlXPathCompOpEvalFilterFirst(ctxt, op, first);
             return (total);
 #endif
         default:
